Description: add table tag at the end of supernets
 adds </table> tag at the end of supernet output when in html mode
Author: Bartosz Fenski <fenio@debian.org>
Origin: Debian
Bug-Debian: http://bugs.debian.org/600251
Forwarded: no
Last-Update: 2012-12-21

--- ipcalc-0.41.orig/ipcalc
+++ ipcalc-0.41/ipcalc
@@ -225,6 +225,9 @@ EOF
    if ($mask1 > $mask2) {
       print "Supernet\n\n";
       supernet($network,$mask1,$mask2);
+      if ($opt_html) {
+         html("</table>\n");
+      }
    }
    if ($opt_html) {
       print << 'EOF';
